locales-src/onboarding.ftl

```diff
@@ -29,6 +29,29 @@ onboarding-join-form-legal = By proceeding, you agree to the <a data-l10n-name="
 onboarding-join-form-continue = Continue
 
 onboarding-start-browsing-button-label = Start Browsing
+onboarding-menu-dismiss =
+    .title = Dismiss
+    .aria-label = Dismiss
+
+## Firefox Sync modal dialog strings.
+
+onboarding-sync-welcome-header = Take { -brand-product-name } with You
+onboarding-sync-welcome-content = Get your bookmarks, history, passwords and other settings on all your devices.
+onboarding-sync-welcome-learn-more-link = Learn more about { -brand-product-name } Accounts
```

Comment:
Per previous conversations, leave this as "Firefox Accounts", don't replace "Firefox".